images/make_osx_icon.sh

```diff
@@ -13,23 +13,26 @@ PNGCRUSH="$(which pngcrush)"
 SVG_FILE="${SCRIPT_ROOT}/OmegaT.svg"
 [ ! -f "$SVG_FILE" ] && echo "Can't find SVG to convert." && exit 1
 
-SIZES="32 64 128 256"
-# 512px version is taken from website:
-# http://briac.net/omegat-logo/
+SIZES="16 32 64 128 256 512 1024"
 
 for SIZE in $SIZES; do
     OUTFILE="${SCRIPT_ROOT}/OmegaT.iconset/icon_${SIZE}x${SIZE}.png"
-    inkscape --export-area-page \
-             --export-width=$SIZE \
-             --export-height=$SIZE \
-             --export-png="$OUTFILE" \
-             --file="$SVG_FILE"
+    inkscape "$SVG_FILE" \
+		--export-type="png" \
+		--export-area-page \
+		--export-width=$SIZE \
+		--export-height=$SIZE \
+		--export-filename="$OUTFILE" \
+
     pngcrush -ow "$OUTFILE"
 done
 
-# The system likes 64x64 to be named 32x32@2x for some reason.
-mv "${SCRIPT_ROOT}/OmegaT.iconset/icon_64x64.png" \
-   "${SCRIPT_ROOT}/OmegaT.iconset/icon_32x32@2x.png"
+cp "${SCRIPT_ROOT}/OmegaT.iconset/icon_32x32.png" "${SCRIPT_ROOT}/OmegaT.iconset/icon_16x16@2x.png"
+cp "${SCRIPT_ROOT}/OmegaT.iconset/icon_64x64.png" "${SCRIPT_ROOT}/OmegaT.iconset/icon_32x32@2x.png"
+cp "${SCRIPT_ROOT}/OmegaT.iconset/icon_128x128.png" "${SCRIPT_ROOT}/OmegaT.iconset/icon_64x64@2x.png"
+cp "${SCRIPT_ROOT}/OmegaT.iconset/icon_256x256.png" "${SCRIPT_ROOT}/OmegaT.iconset/icon_128x128@2x.png"
+cp "${SCRIPT_ROOT}/OmegaT.iconset/icon_512x512.png" "${SCRIPT_ROOT}/OmegaT.iconset/icon_256x256@2x.png"
+mv "${SCRIPT_ROOT}/OmegaT.iconset/icon_1024x1024.png" "${SCRIPT_ROOT}/OmegaT.iconset/icon_512x512@2x.png"
 
 iconutil --convert icns "${SCRIPT_ROOT}/OmegaT.iconset"
```

Comment:
```suggestion
iconutil --convert icns "${SCRIPT_ROOT}/OmegaT.iconset"
# install icnsutil
python -m venv icnsenv
source icnsenv/bin/activate
python -m pip install icnsutil
# print produced icons
icnsutil p "${SCRIPT_ROOT}/OmegaT.icns"

# verify valid format?
icnsutil t "${SCRIPT_ROOT}/OmegaT.icns"

```

Can we use [icnsutils](https://pypi.org/project/icnsutil/) to validate resulted `*.icns` file?

Reply:
With your suggestion, here is what I have:

```
 ./make_osx_icon.sh                                                                                            
  Recompressing IDAT chunks in /Users/suzume/Documents/Repositories/omegat.brandelune/images/OmegaT.iconset/icon_16x16.png
   Total length of data found in critical chunks            =       824
   Best pngcrush method        =   7 (ws 11 fm 0 zl 9 zs 0) =       823
CPU time decode 0.000207, encode 0.000552, other 0.001751, total 0.002529 sec
  Recompressing IDAT chunks in /Users/suzume/Documents/Repositories/omegat.brandelune/images/OmegaT.iconset/icon_32x32.png
   Total length of data found in critical chunks            =      2014
   Best pngcrush method        =   7 (ws 13 fm 0 zl 9 zs 0) =      1973
CPU time decode 0.000624, encode 0.001436, other 0.001554, total 0.003732 sec
  Recompressing IDAT chunks in /Users/suzume/Documents/Repositories/omegat.brandelune/images/OmegaT.iconset/icon_64x64.png
   Total length of data found in critical chunks            =      5095
   Best pngcrush method        =   7 (ws 15 fm 0 zl 9 zs 0) =      5086
CPU time decode 0.001336, encode 0.002847, other 0.001484, total 0.005752 sec
  Recompressing IDAT chunks in /Users/suzume/Documents/Repositories/omegat.brandelune/images/OmegaT.iconset/icon_128x128.png
   Total length of data found in critical chunks            =     13056
   Best pngcrush method        =   7 (ws 15 fm 0 zl 9 zs 0) =     12089
CPU time decode 0.003036, encode 0.009804, other 0.001405, total 0.014439 sec
  Recompressing IDAT chunks in /Users/suzume/Documents/Repositories/omegat.brandelune/images/OmegaT.iconset/icon_256x256.png
   Total length of data found in critical chunks            =     32881
   Best pngcrush method        =   7 (ws 15 fm 0 zl 9 zs 0) =     28790
CPU time decode 0.008542, encode 0.042574, other 0.001716, total 0.053281 sec
  Recompressing IDAT chunks in /Users/suzume/Documents/Repositories/omegat.brandelune/images/OmegaT.iconset/icon_512x512.png
   Total length of data found in critical chunks            =     83171
   Best pngcrush method        =   7 (ws 15 fm 0 zl 9 zs 0) =     68187
CPU time decode 0.025796, encode 0.174426, other 0.001915, total 0.202870 sec
  Recompressing IDAT chunks in /Users/suzume/Documents/Repositories/omegat.brandelune/images/OmegaT.iconset/icon_1024x1024.png
   Total length of data found in critical chunks            =    218630
   Best pngcrush method        =   7 (ws 15 fm 0 zl 9 zs 0) =    169455
CPU time decode 0.091268, encode 0.621755, other 0.002331, total 0.716830 sec
Requirement already satisfied: icnsutil in ./icnsenv/lib/python3.10/site-packages (1.0.1)
File: /Users/suzume/Documents/Repositories/omegat.brandelune/images/OmegaT.icns
  ic12: 5674 bytes, png: 32x32@2x
  ic07: 15278 bytes, png: 128x128
  ic13: 42071 bytes, png: 128x128@2x
  ic08: 42071 bytes, png: 256x256
  ic04: 869 bytes, argb: 16x16
  ic14: 117364 bytes, png: 256x256@2x
  ic09: 117364 bytes, png: 512x512
  ic05: 2650 bytes, argb: 32x32
  ic10: 343649 bytes, png: 512x512@2x
  ic11: 2190 bytes, png: 16x16@2x
  info: 310 bytes, plist: info

File: /Users/suzume/Documents/Repositories/omegat.brandelune/images/OmegaT.icns
OK
```

Reply:
If that is what you expect, then let's go ahead. Do you need me to push anything ?

Reply:
Oh,  it looks good.

Reply:
We now have ic04, ic05, ic07, ic08, ic09, ic10, ic11, ic12, ic13, ic14 icons, 10 icon files.

I hope you can get clear view of icon on finder.

Reply:
<img width="62" alt="Screen Shot 2022-10-02 at 18 07 59" src="https://user-images.githubusercontent.com/28946328/193446780-87458f8d-0640-442e-aaf9-f75f4ded6d28.png">

Reply:
I don't see any difference. But if we agree that it's cute to see a pixelized OmegaT, then I'm fine with it. It's an improvement on having different logos anyway.

Reply:
OK, I'd like to merge here.